community/vote-a-core-developer.md

```diff
@@ -210,7 +210,7 @@ receive a follow-up message with the next steps for
 establishing you as a committer.
 ```
 
-For inviting the PPMC member, the mail template can refer [this](https://incubator.apache.org/guides/ppmc-offer.txt).
```

Comment:
Perhaps change the link also. I'm not sure which one can be as a replacement, though.

Reply:
Yes, I didn’t find the right link, so leave it as it’s in this PR.